h2o-bindings/bin/gen_R.py

```diff
@@ -645,6 +650,14 @@ def help_example_for(algo):
         # See example R code here:
         # http://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/stacked-ensembles.html
         """
+    if algo == "generic":
+        return """\donttest{
+            library(h2o)
+            h2o.init()
+            
+            generic_model <- h2o.genericModel("/path/to/mojo.zip")
```

Comment:
Is this the prefered way to load the mojo model?

Reply:
Yes it is. It is a convenience method over what is generated automatically, where the users needs to import the file with mojo on their own.